storage/spanner/setup.go

```diff
@@ -0,0 +1,20 @@
+package spanner
+
+import (
+	"context"
+	"fmt"
+
+	"cloud.google.com/go/spanner"
+
+	"github.com/bitleak/lmstfy/storage/conf"
+)
+
+func CreateSpannerClient(cfg *conf.SpannerConfig) (*spanner.Client, error) {
+	db := fmt.Sprintf("projects/%s/instances/%s/databases/%s", cfg.Project, cfg.Instance, cfg.Database)
+	spannerClient, err := spanner.NewClient(context.Background(), db)
+	if err != nil {
```

Comment:
Need to set credentials if it's assigned

Reply:
Yep, will add credentials file.